src/main/java/com/google/sps/data/Comment.java

```diff
@@ -26,6 +33,21 @@ public long getID() {
         return id; 
     }
 
+    public static boolean isCommentToxic(String msg) {
```

Comment:
this method should either go in a new file ToxicityDetector.java (or something like that) and take in a String. or this method should be an instance method and just operator on this.comment.

If you have questions about this, feel free to ask.